src/logging_stacktrace.c

```diff
@@ -52,82 +59,86 @@ void getStackAsString(char* destination, size_t size)
         }
     }
 
-    size_t destinationSize = size;
     size_t copied;
 
-    void* stack[TRACE_MAX_STACK_FRAMES];
-
     /*all following function calls are protected by the same SRW*/
     AcquireSRWLockExclusive(&lockOverSymCalls);
 
     uint16_t numberOfFrames = CaptureStackBackTrace(1, TRACE_MAX_STACK_FRAMES, stack, NULL);
-    HANDLE process = GetCurrentProcess();
-
-    SYMBOL_INFO_EXTENDED symbolExtended;
-    SYMBOL_INFO* symbol = &symbolExtended.symbol;
+    if (numberOfFrames == 0)
+    {
+        snprintf(destination, destinationSize, "!CaptureStackBackTrace returned 0 frames");
```

Comment:
>snprintf [](start = 8, length = 8)

(void)? Theoretically it could also fail I guess. Theoretically, 'cause there's not much that should fail.

Reply:
added (void)

---
In reply to: [504394773](https://github.com/Azure/azure-c-logging/pull/18#discussion_r504394773) [](ancestors = 504394773)